scalarflow/charts/data-population/values.yaml

```diff
@@ -0,0 +1,74 @@
+# Default values for data-population.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+dataPopulation:
+  image:
+    repository: ghcr.io/scalar-labs/scalarflow-data-population
+    pullPolicy: IfNotPresent
+    tag: 1.0.0
+  # This will clean the data population job after the job is finished (either completed or failed) after a set number of seconds. "0" means disable this feature (the job is not cleaned).
+  ttlSecondsAfterFinished: 60
+  # Arguments for the scalarflow data population
+  args: []
+  
+  scalardb:
+    config: scalardb.properties
+    configData: ""
+    # In the case the properties file is not used, then the configuration can be done via bellow configs
+    namespace: scalarflow
+    contactPoints: "scalardb-server-envoy"
+    contactPort: 60051
+    username: ""
+    password: ""
+    storage: grpc
+    # If you are not using ScalarDB server, you can leave this value to be empty to use the default consensus commit transaction manager
+    transactionManager: grpc
+
+  configMouthPath: /scalarflow
+
+imagePullSecrets:
+  - name: regcred
+
+nameOverride: ""
+fullnameOverride: ""
```

Comment:
Just a question. Do you have any use cases of `nameOverride` and `fullnameOverride`?
It seems that the ScalarFlow API chart and ScalarFlow Web chart do not support these two parameters.
So, I want to know the reason why the only Data Population chart supports these parameters.

Reply:
These values come from the default template, we can remove them I think. I updated to remove them